blogposts/announcing-sourcegraph-3.10.md

```diff
@@ -0,0 +1,198 @@
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/372226334/de668e24fa?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/372226334/de668e24fa" target="_blank">View on Vimeo</a></p>
+</p>
+
+Sourcegraph users use the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension) to take advantage of go-to-definition and find references on their code hosts. GitLab recognizes the value this brings to developer productivity and has partnered with Sourcegraph to provide this functionality natively to all GitLab users. Read our [feature announcement](https://about.sourcegraph.com/blog/gitlab-integrates-sourcegraph-code-navigation-and-code-intelligence) and [GitLab’s post](https://about.gitlab.com/blog/2019/11/12/sourcegraph-code-intelligence-integration-for-gitlab/) about collaborating in the open.
+
+Soon, GitLab.com users will be able to opt-in to native code intelligence for public repositories by enabling Sourcegraph in their user preferences. This feature is being rolled out incrementally to public projects on GitLab.com, and will first be available on the [GitLab repository](https://gitlab.com/gitlab-org/gitlab).
+
+Users with private GitLab instances or repositories can enable Sourcegraph code intelligence as of GitLab 12.5. Take a look at the [Sourcegraph integration documentation](https://gitlab.com/gitlab-org/gitlab/blob/master/doc/integration/sourcegraph.md) for how to configure it on your instance.
+
+## Create cross-repository search and replace campaigns
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Sourcegraph now supports creating and tracking code-aware search and replace campaigns on GitHub and Bitbucket Server. When running a search and replace campaign, you preview the changes, then generate the corresponding pull requests on your GitHub and/or Bitbucket Server instances. Once created, you can track the PRs to completion in a burndown chart.
+
+Sourcegraph is introducing [Comby syntax](https://comby.dev/#basic-usage) for automation search and replace campaigns. Comby is a code aware search syntax that simplifies finding patterns in code. This makes operations that previously required complex regexp or sed queries much simpler to write. For example, changing how errors are logged througout your code:
```

Comment:
> Comby is a code aware search syntax

Since comby is a tool and not a syntax, maybe "Comby uses" or "supports" or a code aware search syntax? Alt: `Comby syntax is code aware and ...`

> `code-aware`

I think?

> This makes operations that previously required complex regexp or sed queries much simpler to write.

Not so sure about the "previously", it could create the impression that we somehow had this capability before and now changed it (I follow the intended meaning). It might also be a good idea to communicate that this is new? Maybe: `This new capability makes operations that could require complex or awkward regexp queries much simpler to write.`

^ Not super attached, just giving a high level impression. 

> For example, to change how errors are logged throughout your code:

s/changing/to change

Reply:
I referred to it as Comby since that is how the campaign is referenced. Should we be calling this structural search and replace instead? I'm happy to introduce new concepts to users, but want to make sure it's the right one...

Reply:
Updated - let me know if the updated resolves everything for you!